Snowflake.Data/Core/SFSession.cs

```diff
@@ -160,10 +160,17 @@ internal void close()
                 authorizationToken = string.Format(SF_AUTHORIZATION_SNOWFLAKE_FMT, sessionToken)
             };
           
-            var response = restRequester.Post<CloseResponse>(closeSessionRequest);
-            if (!response.success)
+            try
+            {
+                var response = restRequester.Post<CloseResponse>(closeSessionRequest);
+                if (!response.success)
+                {
+                    logger.Debug($"Failed to delete session, error ignored. Code: {response.code} Message: {response.message}");
+                }
+            }
+            catch (Exception ex)

```

Comment:
Swallowing errors on `Close()` could be considered a breaking change if anyone was relying on this behavior.